spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/cache/CacheAutoConfigurationTests.java

```diff
@@ -1007,6 +1018,16 @@ public CacheResolver cacheResolver() {
 
 	}
 
+	@Configuration
+	static class AdditionalCacheManagerAutoConfiguration {
```

Comment:
That's not an auto-configuration and shouldn't be named like that. It shouldn't be added as an auto-config in the test either.